src/plugin/CheckoutBilling.plugin.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/braintree-graphql
+ * @link https://github.com/scandipwa/braintree-graphql
+ */
+
+import CheckoutPayments from 'Component/CheckoutPayments';
+
+export class CheckoutBillingPlugin {
+    aroundRenderPayments(args, callback, instance) {
+        const {
+            isSavePayment,
+            paymentMethods,
+            onPaymentMethodSelect,
+            onPaymentSavedInVaultChange,
+            onStoredPaymentMethodSelect,
+            setLoading,
+            setDetailsStep,
+            shippingAddress
+        } = instance.props;
+
+        if (!paymentMethods.length) {
+            return null;
+        }
+
+        return (
+            <CheckoutPayments
+              isSavePayment={ isSavePayment }
+              setLoading={ setLoading }
+              setDetailsStep={ setDetailsStep }
+              paymentMethods={ paymentMethods }
+              onPaymentMethodSelect={ onPaymentMethodSelect }
+              onPaymentSavedInVaultChange={ onPaymentSavedInVaultChange }
+              onStoredPaymentMethodSelect={ onStoredPaymentMethodSelect }
+              setOrderButtonVisibility={ instance.setOrderButtonVisibility }
+              billingAddress={ shippingAddress }
+              setOrderButtonEnableStatus={ instance.setOrderButtonEnableStatus }
```

Comment:
This will overwrite EVERY PLUGIN that exists on this method. Basically, this payment method in the current variant is not compatible with ANY other payment methods. Please, use `cloneElement` in order to pass additional props to the element returned by `callback(...args)`, if that's what you pursue here.